src/ui/window.ui

```diff
@@ -1,279 +1,153 @@
               </object>
-              <packing>
-                <property name="expand">False</property>
-                <property name="fill">True</property>
-                <property name="position">2</property>
-              </packing>
             </child>
             <child>
-              <object class="GtkButtonBox">
-                <property name="visible">True</property>
-                <property name="can-focus">False</property>
-                <property name="margin-top">20</property>
-                <property name="layout-style">start</property>
+              <object class="GtkBox">
+                <property name="margin-top">10</property>
+                <property name="spacing">10</property>
+                <property name="halign">center</property>
+                <property name="valign">center</property>
                 <child>
                   <object class="GtkLabel">
-                    <property name="visible">True</property>
-                    <property name="can-focus">False</property>
                     <property name="halign">end</property>
-                    <property name="label" translatable="yes">Lossless</property>
+                    <property name="label" translatable="1">Lossless</property>
                     <attributes>
-                      <attribute name="style" value="italic"/>
+                      <attribute name="style" value="italic"></attribute>
                     </attributes>
                   </object>
-                  <packing>
-                    <property name="expand">True</property>
-                    <property name="fill">True</property>
-                    <property name="position">0</property>
-                  </packing>
                 </child>
                 <child>
                   <object class="GtkSwitch" id="toggle_lossy">
-                    <property name="visible">True</property>
-                    <property name="can-focus">True</property>
                     <property name="halign">center</property>
                     <property name="valign">center</property>
                   </object>
-                  <packing>
-                    <property name="expand">True</property>
-                    <property name="fill">True</property>
-                    <property name="position">1</property>
-                  </packing>
                 </child>
                 <child>
                   <object class="GtkLabel">
-                    <property name="visible">True</property>
-                    <property name="can-focus">False</property>
                     <property name="halign">start</property>
-                    <property name="label" translatable="yes">Lossy</property>
+                    <property name="label" translatable="1">Lossy</property>
                     <attributes>
-                      <attribute name="weight" value="bold"/>
+                      <attribute name="weight" value="bold"></attribute>
```

Comment:
In general its better to use css classes from https://flathub.org/apps/details/org.gnome.design.Typography